ndbench-core/src/main/java/com/netflix/ndbench/core/generators/SlidingWindowFlipStringKeyGenerator.java

```diff
@@ -0,0 +1,109 @@
+/*
+ *  Copyright 2016 Netflix, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+package com.netflix.ndbench.core.generators;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+/**
+ * @author vchella
+ */
+public class SlidingWindowFlipStringKeyGenerator implements KeyGenerator<String> {
```

Comment:
If in the future we want to generate Javadocs, it might be a good idea to just describe what `SlidingWindowFlip` does inside the class.